### DISCUSSION TITLE
#Beginner's guide

### DISCUSSION THREAD

**Discussion by RajivRanasingha** [DuajDiaz/Travel-Website#28] [Q&A] [answered]
Why Char array is preferred over String for storing password?

**Comment by Bunny77K9** [comment, marked as answer]
These questions help improve your knowledge of string as a data structure. If you can solve all these String questions without any help then you are in good shape.

If you want to learn more about String and other data structures then I suggest you check out the Data Structure and Algorithm MasterClass Part 1 - 2 on Pluralsight.

**Reply by DuajDiaz** [reply]
Thank for your answer!
